src/meta/src/barrier/mod.rs

```diff
@@ -160,6 +178,28 @@ pub struct GlobalBarrierManager<S: MetaStore> {
     metrics: Arc<MetaMetrics>,
 
     env: MetaSrvEnv<S>,
+
+    wait_recovery: Arc<AtomicBool>,
+
+    wait_build_actor: Arc<AtomicBool>,
+
+    command_ctx_queue: Arc<RwLock<VecDeque<EpochNode<S>>>>,
```

Comment:
Could you please add some doc for these fields?

Reply:
okok